com.github.SubhadeepJasu.pebbles.yaml

```diff
@@ -0,0 +1,47 @@
+app-id: com.github.SubhadeepJasu.pebbles
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+base: io.elementary.BaseApp
+base-version: juno-20.08
+command: com.github.subhadeepjasu.pebbles
+separate-locales: false
+finish-args:
+  - --share=network
+  - --socket=x11
+  - --socket=fallback-x11
+  - --socket=session-bus
+cleanup:
+  - '.la'
+  - '*.a'
+  - /cache
+  - /include
+  - /lib/pkgconfig
+  - /lib/systemd
+  - /lib/valadoc
+  - /lib/valadoc-0.48
+  - /man
+  - /gls
+  - /share/aclocal
+  - /share/devhelp
+  - /share/gir-1.0
+  - /share/gtk-doc
+  - /share/man
+  - /share/vala-0.40
+  - /share/vala-0.48
+  - /share/valadoc
+  - /share/valadoc-0.48
+modules:
+  - name: gls
```

Comment:
typo: I think you meant `gsl`

(it doesn't really change anything to the build to be honest)

Reply:
Good catch!